api/types/constants.go

```diff
@@ -190,6 +190,9 @@ const (
 	// KindMFADevice is an MFA device for a user.
 	KindMFADevice = "mfa_device"
 
+	// KindBillingInformation represents access to cloud billing features
+	KindBillingInformation = "billing_information"
```

Comment:
Nit: Maybe it's just me but I prefer short names unless they compromise meaning, so I'd just name this "billing" (information suffix seems redundant).